pkgs/development/python-modules/craft-application/default.nix

```diff
@@ -30,7 +30,7 @@
 
 buildPythonPackage rec {
   pname = "craft-application";
-  version = "4.8.2";
+  version = "4.9.0";
   pyproject = true;
 
   disabled = pythonOlder "3.10";
```

Comment:
I think this can go away from everywhere now.

Reply:
Removed :)